app/browser/menu.js

```diff
@@ -36,7 +36,9 @@ const menuUtil = require('../common/lib/menuUtil')
 const {getSetting} = require('../../js/settings')
 const locale = require('../locale')
 const {isLocationBookmarked} = require('../../js/state/siteUtil')
-const {isDarwin, isLinux} = require('../common/lib/platformUtil')
+const platformUtil = require('../common/lib/platformUtil')
+const isDarwin = platformUtil.isDarwin()
+const isLinux = platformUtil.isLinux()
```

Comment:
This is the most important part of the PR.  Without this PR, isDarwin and isLinux are always truthy (because they are set to the function itself, not the return value of the function)